README.md

```diff
@@ -16,3 +16,48 @@ If you prefer, you can read all of the website content on GitHub.com instead:
 * [About](about.md)
 
 Otherwise, head over to https://nf-core.github.io and enjoy!
+
+## Contributing to nf-core.github.io
```

Comment:
Maybe easier to just link to the GitHub documentation? https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/

Reply:
well, it worked. I followed the Jekyll documentation on https://jekyllrb.com/ and failed at first.